transport/http/src/main/java/org/kaazing/gateway/transport/http/HttpConnectProcessor.java

```diff
@@ -71,6 +71,7 @@ protected void finishConnect(DefaultHttpSession session) {
             // create HttpRequestMessage
             HttpRequestMessage httpRequest = new HttpRequestMessage();
             httpRequest.setMethod(session.getMethod());
+ 
```

Comment:
You may want to remove this new line. It doesn't add any value as httpRequest is still poplulated with other details.